cloud_vm.md

```diff
@@ -24,12 +24,20 @@ Click "Create Instance" and change the following settings from the default:
 > If you still do not see the container running, check for errors with `sudo journalctl -u konlet-startup`
 
 - Once the climsim container is running, you can get shell access in the climsim container using `docker exec -it <CONTAINER ID> /bin/bash`
+- Update the E3SM repository. There are updates on E3SM codes that are not yet included in the container.
+    - `cd /climsim`
+    - `mv E3SM E3SM_old`
+    - `git clone https://github.com/NVlabs/E3SM.git`
+    - `cd E3SM`
+    - `git config --global url.https://github.com/.insteadOf git@github.com:`
```

Comment:
Why is this needed? Can we just make all the submodules https based?

Reply:
Also, having `--global` might be dangerous.

Reply:
git config --global is needed here for cloud vm. The purpose is to skip setting up ssh keys in the vm. this git config command is not necessary for local machine setup where ssh keys are already set up. I've updated this message in the readme.